dali/pipeline/proto/dali_proto_intern.cc

```diff
@@ -54,6 +54,9 @@ namespace dali {
     intern_->add_bools(val);
   }
 
+  void DaliProtoPriv::add_doubles(const double &val) {
+    intern_->add_doubles(val);
```

Comment:
@JanuszL  do we plan to support doubles?

Reply:
If there is a strong reason then why now.
@klecki , @mzient ?

Reply:
It's a leftover of the initial implem of the op, where I was planning to have `padding_value` as a double. I can remove it

Reply:
what's the conclusion on this topic?